os/fs/procfs/fs_procfslogsave.c

```diff
@@ -141,6 +142,15 @@ static ssize_t logsave_read(FAR struct file *filep, FAR char *buffer, size_t buf
 	return log_dump_read(buffer, buflen);
 }
 
+/****************************************************************************
+ * Name: logsave_write
+ ****************************************************************************/
+
+static ssize_t logsave_write(FAR struct file *filep, FAR const char *buffer, size_t buflen)
+{
+	return log_dump_set_option(buffer, buflen);
```

Comment:
It's difficult to say "option" for start and stop. How about just `log_dump_set`?

Reply:
Modified. Thanks.